met-web/src/components/MetDesignSystem/Navigation/Breadcrumb.tsx

```diff
@@ -0,0 +1,29 @@
+import { Breadcrumbs } from '@mui/material';
+import React from 'react';
+import { BodyText, Link } from '../Typography';
+
+type BreadcrumbProps = {
+    name: string;
+    link?: string;
+};
+
+export const BreadcrumbTrail: React.FC<{ crumbs: BreadcrumbProps[]; smallScreenOnly?: boolean }> = ({
+    crumbs,
+    smallScreenOnly,
+}) => {
+    return (
+        <Breadcrumbs aria-label="breadcrumb" sx={smallScreenOnly ? { display: { xs: 'block', md: 'none' } } : {}}>
```

Comment:
I feel like it'd be a bit more Reactive to have screen size check happen outside the component and have it not render, rather than having it built in and trigger a display:none. Although I get the impulse to have a component be super self-contained.